pkg/reconciler/revision/queueing_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Knative Authors.
+Copyright 2018 The Knative Authors
```

Comment:
Bad push?

Reply:
probably the result of some merge -- it looks like some of the files include a `.` as a part of this section & some do not.  Not sure if there's a pattern or not.